.github/workflows/python-release.yaml

```diff
@@ -126,7 +126,7 @@ jobs:
       - name: Build wheels
         uses: PyO3/maturin-action@v1
         with:
-          target: ${{ matrix.target }}
+          target: ${{ matrix.target }}-apple-darwin
```

Comment:
AFAIK, `x86_64` and `aarch64` arent valid targets, they should be they fully qualified ones `aarch64-apple-darwin`

https://www.maturin.rs/config#target-specific-maturin-options

Reply:
Yea, I noticed this in #2556 as well.

Reply:
is this a recent change?